scripts/dumps/lib/serialize_entity_in_turtle.coffee

```diff
@@ -5,16 +5,31 @@ properties = __.require 'controllers', 'entities/lib/properties/properties_value
 { yellow } = require 'chalk'
 
 module.exports = (entity)->
-  { _id } = entity
+  { _id, _rev, created, updated, type, redirect } = entity
+
+  if type isnt 'entity' or redirect then return ''
 
   text = "inv:#{_id} a wikibase:Item ;"
 
+  dateCreated = new Date(created).toISOString()
+  text += """\n  schema:dateCreated "#{dateCreated}"^^xsd:dateTime ;"""
+
+  dateModified = new Date(updated).toISOString()
+  text += """\n  schema:dateModified "#{dateModified}"^^xsd:dateTime ;"""
+
+  # Using the _rev to deduce the version might give a version number
+  # higher than the real number of patches, but that's better than nothing
+  version = parseInt _rev.split('-')
```

Comment:
`parseInt` indeed according to the manual "If parseInt encounters a character that is not a numeral in the specified radix, it ignores it and all succeeding characters and returns the integer value parsed up to that point." its not it's primary function tho. why not go with a `_rev.split('-')[0]` ?